package.json

```diff
@@ -63,10 +69,11 @@
         "faker": "^4.1.0",
         "jest": "^26.0.1",
         "jsdoc": "^3.6.4",
+        "papaparse": "^5.3.0",
+        "prytaneum-typings": "https://github.com/kadey001/prytaneum-typings#v1.0.0-alpha.69",
```

Comment:
I updated prytaneum typings with your PR and you should update this to the latest release.